compiler/vm/vm.nim

```diff
@@ -1550,8 +1550,7 @@ proc rawExecute(c: var TCtx, t: var VmThread, pc: var int): YieldReason =
     of opcEqNimNode:
       decodeBC(rkInt)
       regs[ra].intVal =
-        ord(exprStructuralEquivalent(regs[rb].nimNode, regs[rc].nimNode,
-                                     strictSymEquality=true))
+        ord(exprStructuralEquivalentStrictSymAndComm(regs[rb].nimNode, regs[rc].nimNode))
```

Comment:
Could you add a test for this bugfix? (`NimNode` equality now considers comments)

Alternatively, you also use the version ignoring comments and add a `TODO`.

Reply:
It did consider comment equality before this change, because `exprStructuralEquivalent` prior to this PR checked `nkCommentStmt` nodes for equality.

Reply:
Oh, you're right, my mistake.